web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -212,172 +218,202 @@ details.captcha > div {
-    }
-    .distance-container {
-      margin: 0;
-    }
-    .form-item > input[type="text"] {
-      max-width: 300px;
-    }
+.field--type-text, .field--type-string, .field--type-datetime, .field--type-text-long {
+  input, textarea {
+    background-color: #F6F4F4;
+    text-indent: 10px;
+  }
+}
+
+tr {
+    border-bottom: 1px solid #FF904B;
+    background-color: #FFFFFF;
 
-    > div {
-      vertical-align: middle;
+    &:hover {
+      background-color: #FFFFFF;
     }
-  }
-  /**
-  input[type="checkbox"] {
-     appearance: none;
-     -webkit-appearance: none;
-     border: 2px solid #CACACA;
-     border-radius: 2px;
-     width: 18px;
-     height: 18px;
-     vertical-align: middle;
-   }
-
-  input[type="checkbox"]:hover,
-  input[type="checkbox"]:focus {
-    border-color: #FF904B;
-  }
+}
+
+div.paragraphs-dropbutton-wrapper {
+  font-size: 80%;
+}
 
-  input[type="checkbox"]:checked {
-    background-color: #FF904B;
-    background-image: url(../assets/icon/tick_white.png);
-    background-repeat: no-repeat;
+ul.select2-selection__rendered {
+  border:  1px solid rgba(0, 0, 0, 0.12);
+  padding-left: 10px !important;
+  background: #F6F4F4;
+
+  &:focus-within {
     border-color: #FF904B;
   }
+}
 
-  input[type="checkbox"]:checked:hover,
-  input[type="checkbox"]:checked:focus {
-    border-color: #CACACA;
-  }*
+.select2-container--cj_material span.select2-selection.select2-selection--multiple {
+  padding-top: 0;
+  padding-bottom: 0;
+  border-bottom: none !important;
+  box-shadow: none !important;
+}
 
-  .form-item.search-wrapper {
-    border: solid 1px #FF904B;
-    border-radius: 4px;
-    padding-left: 40px;
-    margin-left: auto;
-    margin-right: auto;
-    width: fit-content;
-    position: relative;
-    overflow: hidden;
-  }
-  .form-item.search-wrapper:before {
+.select2-container--cj_material .select2-results__option {
+  cursor: pointer;
+  clear: both;
+  color: #FFFFFF;
+  background-color: #49BCFB;
+  line-height: 0.9rem;
+  text-align: left;
+  text-transform: none;
+  font-size: 1.2rem;
+  display: block;
+  padding: 1rem;
+}
+
+.select2-dropdown {
+  margin-top: -23px;
+  width: 1120px;
+  max-width: 30%;
+  background: transparent;
+}
+
+.proximity-filter-container .form-item input[type=text] {
+  display: inline-block;
+}
+
+form {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+summary {
+  border-bottom: 6px solid #FF904B;
+  background-color: #FFFFFF;
+  color: #FF904B;
+  font-family: Montserrat;
+  font-weight: 800;
+  font-size: 23px;
+  margin: 40px -40px 20px;
+}
+
+.divider-top:before {
+  content: "";
+  background: #FF904B;
+  height: 6px;
+  position: absolute;
+  left: -40px;
+  top: 0px;
+  width: 1200px;
+}
+
+div.details-wrapper {
+  padding-left: 0px;
+  padding-right: 0px;
+}
+
+span.field-prefix {
+  padding-right: 5px;
+  font-size: 20px;
+}
+
+div.cke_focus {
+  border: solid 2px #FF904B;
+}
+
+div#edit-publish-date-wrapper, div#edit-application-deadline-wrapper {
```

Comment:
Any chance of making these class based rather than #id?

Reply:
Resolved to discuss putting a Span around Descriptions in markup at a later date